src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/BookingContact.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Describes the details of a booking contact")
+data class BookingContact(
+  @Schema(description = "Describes the internal id of the video booking", example = "123")
+  val videoBookingId: Long,
+
+  @Schema(description = "Describes the contact type (OWNER, PRISON, PROBATION, COURT, THIRD_PARTY)", example = "PRISON")
+  val contactType: String,
```

Comment:
Would suggest an enum.  If not then perhaps add allowableValues = ["OWNER" ... ] in the description.

Reply:
Done